.github/workflows/promote.yml

```diff
@@ -12,10 +15,7 @@ jobs:
         uses: levibostian/action-promote-semantic-release@v1
         with:
           sequence: "develop,alpha,beta,main"
-          githubToken: ${{ secrets.WRITE_ACCESS_BOT_TOKEN }}
-          # Find private email address that github uses for CI account at: https://github.com/settings/emails
-          gitName: "Ami CI"
-          gitEmail: "21171259+ami-ci@users.noreply.github.com"
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
In our call together, we talked about how you were experiencing issues with the CI server not being able to make git commits. By removing `gitName` and `gitEmail`, doesn't that mean that we have this problem again?